frontend/components/Players/PlayerManager.vue

```diff
@@ -254,6 +254,7 @@ export default Vue.extend({
     return {
       showFullScreenOverlay: false,
       fullScreenOverlayTimer: null as number | null,
+      clickTimer: null as number | null,
```

Comment:
Why we need this?

Reply:
To prevent dblclick from also firing single click event. So basically, If we do not get another click during that time span, we're firing pause, otherwise fullscreen.

Reply:
@tmneth But have you tested without it? Which problems did you encounter without it? 

Browser's double click event (which you're using) should already handle all of that for you.

https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
@ferferga I have tried to use dblclick and click without a timeout, however, in such case the click handler is triggered when i doubleclick. So doubleclick would both pause and trigger fullscreen at the same time.